File: Persistencia_Cpp/src/Course.cpp
#include "../include/Course.hpp"

// Constructor
Course::Course()
{

}

// Setters
void Course::setIdentifier(int identifier)
{
    m_identifier = identifier;
}
void Course::setName(string name)
{
    m_name = name;
}
void Course::setSurname(string surname)
{
    m_surname = surname;
}

// Getters
int Course::getIdentifier() const
{
    return m_identifier;
}
string Course::getName() const
{
    return m_name;
}
string Course::getSurname() const
{
    return m_surname;
}

// Destructor
Course::~Course()
{

}

File: Persintencia_Python/src/Course.py
class Course:
    def __init__(self):
        self.m_identifier = 0
        self.m_name = ''
        self.m_surname = ''
        
    # Setters
    def setIdentifier(self, identifier):
        self.m_identifier = identifier
        
    def setName(self, name):
        self.m_name = name
        
    def setSurname(self, surname):
        self.m_surname = surname
        
    # Getters
    def getIdentifier(self):
        return self.m_identifier
        
    def getName(self):
        return self.m_name
        
    def getSurname(self):
        return self.m_surname
        
    def __del__(self):
        pass